playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,43 @@
+filebeat:
+  # List of prospectors to fetch data.
+  prospectors:
+    # Each - is a prospector. Below are the prospector specific configurations
+    -
+      paths:
+        - /var/log/syslog
+        - "/var/log/mail.*"
+        - "/var/log/kern.*"
+        - "/var/log/news.*"
+        - "/var/log/*.emerg"
+        - "{{ elasticsearch_log_dir }}/*"
+        - "{{ nginx_log_dir }}/*.log"
+        - "{{ nginx_log_dir }}/*.err"
+        - "{{ COMMON_LOG_DIR }}/tracking/*.log"
+        - "{{ COMMON_LOG_DIR }}/cms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/xqueue/edx.log"
+        - "{{ COMMON_LOG_DIR }}/devpi/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/notifier/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/certs/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/tracking/tracking.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/supervisor/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/nginx/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/lms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/cms/edx.log"
+        - "{{ COMMON_LOG_DIR }}/ecommerce/*"
```

Comment:
What was the filename?

This should probably be something of the form:

`"{{ COMMON_LOG_DIR }}/ecommerce/WHATEVER_THE_FILENAME_WAS.log"`